pom.xml

```diff
@@ -63,6 +63,11 @@
             <artifactId>guava</artifactId>
             <version>16.0.1</version>
         </dependency>
+        <dependency>
+            <groupId>org.roaringbitmap</groupId>
+            <artifactId>RoaringBitmap</artifactId>
+            <version>0.5.18</version>
```

Comment:
Is there any reason to include this? It looks like you're just using `org.roaringbitmap.IntIterator`, which is a really small interface.